code/modules/mob/living/carbon/human/species/station/station.dm

```diff
@@ -258,7 +258,7 @@
 		/datum/unarmed_attack/diona
 	)
 	//primitive_form = "Nymph"
-	slowdown = 7
+	slowdown = 5
```

Comment:
So, you're speeding up dionae significantly?

Reply:
>significantly
no

Reply:
It is significantly, and nowhere in the pr or github is stating said change.

Reply:
in 100% light, the dionaea will move at a slowdown of 5. At 0% light, dionaea will move at a slowdown of 7.5. Only a completely naked Dionaea in full lighting will move at 5 movespeed delay. If a Dionaea is wearing a rolled down jumpsuit in 100% light exposure, then a Dionaea would move at roughly 6.5 slowdown.